feature/main/src/main/java/com/readme/android/main/view/MoreBottomSheetDialog.kt

```diff
@@ -45,26 +43,36 @@ class MoreBottomSheetDialog : BottomSheetDialogFragment() {
     }
 
     private fun observeData() {
-        viewModel.isMyFeed.flowWithLifecycle(viewLifecycleOwner.lifecycle)
-            .onEach {
-                with(binding) {
-                    isMyFeed = it
-                    when (it) {
-                        true -> {
-                            tvAction.setOnClickListener {
-                                // TODO 삭제하기 로직 넣기
-                                dismiss()
-                            }
-                        }
-                        else -> {
-                            tvAction.setOnClickListener {
-                                // TODO 신고하기 로직 넣기
-                                dismiss()
-                            }
-                        }
-                    }
-                }
-            }.launchIn(viewLifecycleOwner.lifecycleScope)
+        when (isMyFeed) {
+            true -> binding.tvAction.setOnClickListener {
+                // TODO 삭제하기 로직 넣기
+                dismiss()
+            }
+            false -> binding.tvAction.setOnClickListener {
+                dismiss()
+                sendMail()
+            }
+        }
```

Comment:
관찰되지않는 데이터라면 우선 해당 함수에서 로직을 빼도 될 것 같고,

```suggestion
        binding.tvAction.setOnClickListener {
         when (isMyFeed) {
            true -> { }
            false -> {
                sendMail()
            }
         }
         dismiss()
        }
```
이렇게 바꿔도 되지않을까요?

Reply:
오 좋네요~~ 함수 네이밍도 수정할게요!!

Reply:
이렇게 수정했습니다!

``` kotlin
    private fun onTvActionClickListener() {
        binding.tvAction.setOnClickListener{
            when (isMyFeed) {
                true -> {}// TODO 삭제하기 로직 넣기
                false -> sendMail()
            }
            dismiss()
        }
    }
```